addons/binding/org.openhab.binding.systeminfo/OSGI-INF/SysteminfoHandlerFactory.xml

```diff
@@ -1,20 +1,20 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!--
-
-    Copyright (c) 2010-2017 by the respective copyright holders.
-
-    All rights reserved. This program and the accompanying materials
-    are made available under the terms of the Eclipse Public License v1.0
-    which accompanies this distribution, and is available at
-    http://www.eclipse.org/legal/epl-v10.html
-
--->
-<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="binding.systeminfo">
-
-   <implementation class="org.openhab.binding.systeminfo.internal.SysteminfoHandlerFactory"/>
-
-   <service>
-      <provide interface="org.eclipse.smarthome.core.thing.binding.ThingHandlerFactory"/>
-   </service>
-
-</scr:component>
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2010-2017 by the respective copyright holders.
+
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="binding.systeminfo">
+
+   <implementation class="org.openhab.binding.systeminfo.internal.SysteminfoHandlerFactory"/>
+
+   <service>
+      <provide interface="org.eclipse.smarthome.core.thing.binding.ThingHandlerFactory"/>
+   </service>
+   <reference bind="bindSystemInfo" cardinality="1..1" interface="org.openhab.binding.systeminfo.internal.model.SysteminfoInterface" name="SysteminfoInterface" policy="static" unbind="unbindSystemInfo"/>


```

Comment:
There is some strange inconsistent line ending character at the end of this line.